Base/Python/slicer/util.py

```diff
@@ -641,6 +641,43 @@ def array(pattern = "", index = 0):
 
 
 #
+# Qt
+#
+
+class QtWidgetMixin(object):
+  def __init__(self):
+    super(QtWidgetMixin, self).__init__()
+    self.Widget = None
```

Comment:
Do you mean ```self.Widget``` ? This is used in the get() method. But yes, it is left to the user to be set.
(See my comment about the ```get()``` method).